CHANGELOG/CHANGELOG-1.6.md

```diff
@@ -0,0 +1,114 @@
+- Remove --k8s-1.15 flag from hack/generate-manifest.sh. ([#3350](https://github.com/antrea-io/antrea/pull/3350), [@antoninbas])
+- Remove unnecessary CRDs and RBAC rules from Multi-cluster manifest. ([#3491](https://github.com/antrea-io/antrea/pull/3491), [@luolanzone])
+- Update label and image repo of antrea-mc-controller to be consistent with antrea-controller and antrea-agent. ([#3266](https://github.com/antrea-io/antrea/pull/3266) [#3466](https://github.com/antrea-io/antrea/pull/3466), [@luolanzone])
+- Add clusterID annotation to ServiceExport/Import resources. ([#3359](https://github.com/antrea-io/antrea/pull/3359), [@luolanzone])
+- Do not log error when Service for Endpoints is not found to avoid log spam. ([#3256](https://github.com/antrea-io/antrea/pull/3256), [@tnqn])
+- Ignore Services of type ExternalName for NodePortLocal feature. ([#3114](https://github.com/antrea-io/antrea/pull/3114), [@antoninbas])
+- Add powershell command replacement in the Antrea Windows documentation. ([#3264](https://github.com/antrea-io/antrea/pull/3264), [@GraysonWu])
+
+### Fixed
+
+- Add userspace ARP/NDP responders to fix Egress and ServiceExternalIP support for IPv6 clusters. ([#3318](https://github.com/antrea-io/antrea/pull/3318), [@hty690])
+- Fix incorrect results by `antctl get networkpolicy` when both Pod and Namespace are specified. ([#3499](https://github.com/antrea-io/antrea/pull/3499), [@Dyanngg])
+- Fix IP leak issue when AntreaIPAM is enabled. ([#3314](https://github.com/antrea-io/antrea/pull/3314), [@gran-vmv])
+- Fix error when dumping OVS flows of a NetworkPolicy via `antctl get ovsflows`. ([#3335](https://github.com/antrea-io/antrea/pull/3335), [@jainpulkit22])
+- Fix IPsec encryption for IPv6 overlays. ([#3155](https://github.com/antrea-io/antrea/pull/3155), [@antoninbas])
+- Add ignored interfaces names when getting interface by IP to fix NetworkPolicyOnly mode in AKE. ([#3219](https://github.com/antrea-io/antrea/pull/3219), [@wenyingd])
+- Fix duplicate IP case for NetworkPolicy. ([#3467](https://github.com/antrea-io/antrea/pull/3467), [@tnqn])
+- Don't delete the routes which are added for the peer IPv6 gateways on Agent startup. ([#3336](https://github.com/antrea-io/antrea/pull/3336) [#3490](https://github.com/antrea-io/antrea/pull/3490), [@Jexf] [@xliuxu])
+- Fix pkt mark conflict between HostLocalSourceMark and SNATIPMark. ([#3430](https://github.com/antrea-io/antrea/pull/3430), [@tnqn])
+- Unconditionally sync CA cert for Controller webhooks to fix Egress support when AntreaPolicy is disabled. ([#3421](https://github.com/antrea-io/antrea/pull/3421), [@antoninbas])
+- Fix inability to access NodePort in particular cases. ([#3371](https://github.com/antrea-io/antrea/pull/3371), [@hongliangl])
+- Fix ipBlocks referenced in nested ClusterGroup were not processed correctly. ([#3383](https://github.com/antrea-io/antrea/pull/3383), [@Dyanngg])
```

Comment:
pedant nit: "were" in the sentence above seems redundant

Reply:
I also thought this is not needed, added it according Jianjun's [comment](https://github.com/antrea-io/antrea/pull/3531#discussion_r836702152). Removed it now. @jianjuns please let us know if you believe it's needed.